lib/std/math.zig

```diff
@@ -44,27 +43,13 @@ pub const rad_per_deg = 0.017453292519943295769236907684886127134428718885417254
 /// 180.0/pi
 pub const deg_per_rad = 57.295779513082320876798154814105170332405472466564321549160243861;
 
-pub const floatExponentBits = float.floatExponentBits;
-pub const floatMantissaBits = float.floatMantissaBits;
-pub const floatFractionalBits = float.floatFractionalBits;
-pub const floatExponentMin = float.floatExponentMin;
-pub const floatExponentMax = float.floatExponentMax;
-pub const floatTrueMin = float.floatTrueMin;
-pub const floatMin = float.floatMin;
-pub const floatMax = float.floatMax;
-pub const floatEps = float.floatEps;
-pub const floatEpsAt = float.floatEpsAt;
-pub const inf = float.inf;
-pub const nan = float.nan;
-pub const snan = float.snan;
```

Comment:
It'd probably be good to deprecate these with `@compileError`s first.

Reply:
0.13 is already tagged